src/main/car/functionality/DealershipUI

```diff
@@ -0,0 +1,131 @@
+import javax.swing.*;
+
+public class DealershipUI {
+	static String dealerID = "";
+	
+	private static void makeDealerGUI() {
+	    //dealer information screen
+		JFrame dealerDetails = new JFrame("ICS372 Group 4 Assignment 2");
+		dealerDetails.setLayout(new GridLayout(3,1,0,10));
+		JPanel Options = new JPanel();
+		Options.setLayout(new GridLayout(5,3,10,10));
+		JPanel specificVehicle = new JPanel();
+		specificVehicle.setLayout(new GridLayout());
+		JLabel ID = new JLabel("Dealer: ID here");
+		JLabel Name = new JLabel("Name: ");
+		JLabel Renting = new JLabel("Renting: ");
+		JLabel Acquisition = new JLabel("Vehicle Acquisition: ");
+		JLabel Transfer = new JLabel("Transfer Inventory to: ");
+		JLabel Export = new JLabel("Export: ");
+		JTextField enterName = new JTextField("Dealer Name here");
+		JTextField transferInventory = new JTextField("Receiving Dealer ID here");
+		
+		JButton changeName = new JButton("save");
+		changeName.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
```

Comment:
after getting a specific dealer through the search, you should be able to directly do 
dealer.setName( enterName.getText() ) or something similar.

Reply:
This should be easy to implement with the addition of the parameter in the construction of this GUI to target the correct dealer. This also includes the other JButtons that will be messing with the attributes of the dealer

Reply:
Added function to all buttons in the class. Let me know if I passed over one. Still working on how to transition to Mike's frame in the specificVehicleScreen button at the time of this writing